lib/datasource/docker/__snapshots__/index.spec.ts.snap

```diff
@@ -30,7 +29,6 @@ Array [
   Object {
     "headers": Object {
       "accept-encoding": "gzip, deflate",
-      "authorization": "Basic c29tZS11c2VybmFtZTpzb21lLXBhc3N3b3Jk",
```

Comment:
I'm not an expert with `Docker` datasource, but looks everything looks okay as far as `accept` and `authorization` headers aren't needed here

Reply:
I also wondered why the header was missing actually

Reply:
Accept and content-type is not required for basic registry login check.

Because of the test hostRule mock the authorization header was always send.